README.md

```diff
@@ -1,5 +1,12 @@
 # Variational-DL
-### Contributors 
+
+#### Variational Deep Learning is a method of deep learning where we use Neural Networks to generate data, instead of drawing conclusions from it.
```

Comment:
Don't put H4 for the content part. Please do the changes in both the readmes.

Reply:
Resolved the issue in all files.